graylog2-web-interface/src/components/pipelines/ProcessingTimelineComponent.jsx

```diff
@@ -95,21 +113,21 @@ const ProcessingTimelineComponent = createReactClass({
 
     return (
       <tr key={pipeline.id}>
-        <td className="pipeline-name">
+        <PipelineName>
```

Comment:
I would add `Td` in the name of this component to make it clearer what it renders `PipelineNameTd` without scrolling up and down. Same with `StreamList` below.

Reply:
Done and done!